creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -69,35 +71,19 @@ module.exports = async function ({
     logger.error(error.message)
   }
 
-  // Determine if new sync request needs to be enqueued
-  let getNewOrExistingSyncReqError
-  if (syncReqToEnqueueResp) {
-    const {
-      userWallet,
-      secondaryEndpoint,
-      syncMode: syncModeResp
-    } = syncReqToEnqueueResp
-    const { syncReqToEnqueue, duplicateSyncReq } = getNewOrExistingSyncReq({
-      userWallet,
-      secondaryEndpoint,
-      primaryEndpoint: thisContentNodeEndpoint,
-      syncType,
-      syncMode: syncModeResp
-    })
-    if (!_.isEmpty(duplicateSyncReq)) {
-      getNewOrExistingSyncReqError = {
-        message:
-          'Additional sync request was required but not able to be enqueued due to a duplicate',
-        duplicateSyncReq
-      }
-    } else if (!_.isEmpty(syncReqToEnqueue)) {
-      jobsToEnqueue = { [QUEUE_NAMES.STATE_RECONCILIATION]: [syncReqToEnqueue] }
+  // Enqueue a new sync request if one needs to be enqueued and we haven't retried too many times yet
+  if (
+    !_.isEmpty(syncReqToEnqueue) &&
+    attemptNumber < MAX_ISSUE_SYNC_JOB_ATTEMPTS
+  ) {
```

Comment:
we should log when max attempts exceeded? and potentially associated metric (latter can be separate if not easy)

Reply:
do you mean like a metric to track avg number of times a job is retried? that does sound useful. in the interest of time I just changed it to log for now

Reply:
log is good, and meant metric for when issueSyncRequest fails due to max attempts exceeded